t/api/08-jobtemplates.t

```diff
@@ -516,15 +517,30 @@ is_deeply(['opensuse', 'opensuse test'], [sort keys %$yaml], 'YAML of all groups
 $t->get_ok('/api/v1/job_templates_scheduling/' . $opensuse->id)->status_is(200);
 $yaml = $t->tx->res->body;
 is($t->tx->res->body, $yaml, 'No document start marker by default');
-$yaml = load_yaml(string => $t->tx->res->body);
+$yaml = load_yaml(string => $t->tx->res->json);
 is_deeply($t->app->validate_yaml($yaml, $schema_filename, 1), [], 'YAML of single group is valid');
 my $yaml2 = load_yaml(file => "$FindBin::Bin/../data/08-opensuse-2.yaml");
 is_deeply($yaml, $yaml2, 'YAML for opensuse group') || diag explain $t->tx->res->body;
 
-$t->get_ok('/api/v1/job_templates_scheduling/' . $opensuse->id)->status_is(200)
-  ->content_type_is('text/yaml;charset=UTF-8');
-is_deeply(load_yaml(string => $t->tx->res->body), $yaml, 'Test suite with unicode characters encoded correctly')
-  || diag explain $t->tx->res->body;
+subtest 'content-type' => sub {
+    $t->get_ok('/api/v1/job_templates_scheduling/' . $opensuse->id, $accept_yaml)->status_is(200)
+      ->content_type_is('text/yaml;charset=UTF-8');
+    is_deeply(load_yaml(string => $t->tx->res->body),
+        $yaml, '[Accept: text/yaml] Test suite with unicode characters encoded correctly')
+      || diag explain $t->tx->res->body;
+
+    $t->get_ok('/api/v1/job_templates_scheduling/' . $opensuse->id, {Accept => 'application/json'})->status_is(200)
+      ->content_type_is('application/json;charset=UTF-8');
+    is_deeply(load_yaml(string => $t->tx->res->json),
+        $yaml, '[Accept: application/json] Test suite with unicode characters encoded correctly')
+      || diag explain $t->tx->res->body;
+
+    $t->get_ok('/api/v1/job_templates_scheduling/' . $opensuse->id, {Accept => '*/*'})->status_is(200)
```

Comment:
What about the case where you don't specify a content type explicitly?

Reply:
That's the default (`application/json`) and used in all other requests, but I can add a test for this here, sure.